From 3e1a58180eddae5e450afb60ce1883777ea62d03 Mon Sep 17 00:00:00 2001
From: Wu Guoxing <b02248@freescale.com>
Date: Wed, 6 May 2009 10:25:15 +0800
Subject: [PATCH] ENGR00112203-1 mx35:CONFIG_FORCE_MAX_ZONEORDER Support

CONFIG_FORCE_MAX_ZONEORDER Support For mx35.
this will let user to change the MAX_ORDER value in menuconfig

Signed-off-by: Wu Guoxing <b02248@freescale.com>
---
 arch/arm/mach-mx35/Kconfig |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx35/Kconfig b/arch/arm/mach-mx35/Kconfig
index f00de2d..9d1dbcd 100644
--- a/arch/arm/mach-mx35/Kconfig
+++ b/arch/arm/mach-mx35/Kconfig
@@ -1,6 +1,10 @@
 menu "MX35 Options"
 	depends on ARCH_MX35
 
+config FORCE_MAX_ZONEORDER
+    int "MAX_ORDER"
+    default "11"
+
 config MX35_OPTIONS
 	bool
 	default y
-- 
1.5.4.4

